docs/4.operation/4.infographics.md

```diff
@@ -0,0 +1,62 @@
+## FIWARE Lab Infographic
+
+FIWARE Lab Infographics and Status Pages are simple but important
+services to allow users to:
+
+-   Know in an intuitive way the infrastructure capacities made
+    available by FIWARE Lab infrastructure;
+
+-   Monitor current status of infrastructure services and know about any
+    issue in any node of FIWARE Lab infrastructure.
+
+While the information on infrastructure capacities is more related to
+marketing, the one on services status is extremely important to support
+Developers and Federation Managers operations. The production URL of the
+Infographics is the following:
+
+-   <http://infographic.lab.fiware.org/>
+
+-   <http://status.lab.fi-ware.org/>
+
+This component provides capacity information about the different
+OpenStack regions and status information of infrastructure services. The
+Federation Monitor component exposes data on the capacity and status of
+FIWARE Lab infrastructure through a set of RESTful API that Infographics
+and Status Pages calls in order to get that data.
+
+For more details, it is recommended to refer to the main documentation
+included in the software repository:
+<https://github.com/SmartInfrastructures/fi-lab-infographics>.
+
+Figure below shows the main page of the Infographics tool. It is comprised
+of:
+
+-   a user’s section: it shows the total number of users as sum of Basic
+    Users, Trial Users and Community Users;
+
+-   a map that shows all nodes;
+
+-   six interactive tabs that display different data;
```

Comment:
display different data (number of regions, total number of cores, RAM, and HDD, total number of IPs and the total number of virtual machines currently deployed);